src/responseBuilderHandler.js

```diff
@@ -2,15 +2,16 @@
 
 var ResponseBuilder = require('./ResponseBuilder');
 
-module.exports = function(promiseReturningHandlerFn, request, cb) {
+module.exports = function(promiseReturningHandlerFn, request, cb, CustomRespBuilderClass) {
    promiseReturningHandlerFn()
       .then(function(respBuilder) {
          // eslint-disable-next-line no-console
          console.log('completed with %s millis left', request.getContext().getRemainingTimeInMillis());
          cb(undefined, respBuilder.toResponse(request));
       })
       .catch(function(err) {
-         var respBuilder = new ResponseBuilder(request).error();
+         var RB = CustomRespBuilderClass || ResponseBuilder,
+             respBuilder = new RB(request).error();
```

Comment:
Good catch. I think that was from the initial (possibly uncommitted) version of this code, where the response builder held a reference to the request. I removed it now. Thanks!